src/components/masters/program/ProgramPage.jsx

```diff
@@ -118,10 +118,10 @@ class ProgramPage extends Component {
                   </main>
                 </>
               ) : (
-                this.renderError()
-              )}
+                  this.renderError()
```

Comment:
Are these new indents valid? It looks like the previous indentation was correct.

Reply:
Stupid prettier. Nice catch -- Even the linter didn't seem to care about that one.